source/_components/somfy_mylink.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "Somfy MyLink"
+description: "Instructions on how to integrate Somfy MyLink devices with Home Assistant."
+date: 2018-10-20 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Hub
+ha_release: 0.XX
+ha_iot_class: "Assumed State"
+---
+
```

Comment:
One empty line to many. Please remove one.

Reply:
Have removed in new commit